first_validation.py

```diff
@@ -48,11 +50,23 @@ def check_hashes(hash_fname):
     # Directory containing hash filenames file.
     data_dir = hash_pth.parent
     # Read in text for hash filename
+    text_file = hash_pth.read_text()
     # Split into lines.
+    lines = text_file.splitlines()
     # For each line:
+    for line in lines:
         # Split each line into expected_hash and filename
+        a = line.split('  ')
```

Comment:
Or just use the default behavior, to split at one or multiple spaces,  as in:

```suggestion
        a = line.split()
```

Reply:
You could also separate the values with [sequence unpacking](https://textbook.nipraxis.org/sequence_unpacking), as in:

```suggestion
        exp_hash, filename = line.split()
```